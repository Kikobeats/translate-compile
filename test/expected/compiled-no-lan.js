var angTranslations = {"enUs":{"MENU":{"CART":{"EMPTY":"Empty Cart","CHECKOUT":"Checkout"},"USER":{"LABEL":"User","DROPDOWN":{"EDIT":"Edit","LOGOUT":"Logout"}}},"COUNTRY":{"USA":{"STATES":{"CALIFORNIA":{"NAME":"California"},"NEW_YORK":{"NAME":"New York"},"NORTH_CAROLINA":{"NAME":"North Carolina"}}}}},"ptBr":{"MENU":{"CART":{"EMPTY":"Esvaziar Carrinho","CHECKOUT":"Fechar Pedido"},"USER":{"LABEL":"Usuário","DROPDOWN":{"EDIT":"Editar","LOGOUT":"Sair"}}},"COUNTRY":{"USA":{"STATES":{"CALIFORNIA":{"NAME":"Califórnia"},"NEW_YORK":{"NAME":"Nova Iorque"},"NORTH_CAROLINA":{"NAME":"Carolina Do Norte"}}}}},"esEs":{"MENU":{"CART":{"EMPTY":"Vaciar Carrito","CHECKOUT":"Realizar Pedido"},"USER":{"LABEL":"Usuario","DROPDOWN":{"EDIT":"Editar","LOGOUT":"Finalizar la Sesión"}}},"COUNTRY":{"USA":{"STATES":{"CALIFORNIA":{"NAME":"California"},"NEW_YORK":{"NAME":"Nueva York"},"NORTH_CAROLINA":{"NAME":"Carolina del Norte"}}}}}}