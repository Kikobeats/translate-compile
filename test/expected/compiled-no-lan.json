{"enUs":{"MENU":{"CART":{"EMPTY":"Empty Cart","CHECKOUT":"Checkout"},"USER":{"LABEL":"User","DROPDOWN":{"EDIT":"Edit","LOGOUT":"Logout"}}},"COUNTRY":{"USA":{"STATES":{"CALIFORNIA":{"NAME":"California","ACRONYM":"CA"},"NEW_YORK":{"NAME":"New York","ACRONYM":"NY"},"NORTH_CAROLINA":{"NAME":"North Carolina","ACRONYM":"NC"}}}}},"ptBr":{"MENU":{"CART":{"EMPTY":"Esvaziar Carrinho","CHECKOUT":"Fechar Pedido"},"USER":{"LABEL":"Usuário","DROPDOWN":{"EDIT":"Editar","LOGOUT":"Sair"}}},"COUNTRY":{"USA":{"STATES":{"CALIFORNIA":{"NAME":"Califórnia","ACRONYM":"CA"},"NEW_YORK":{"NAME":"Nova Iorque","ACRONYM":"NY"},"NORTH_CAROLINA":{"NAME":"Carolina Do Norte","ACRONYM":"NC"}}}}},"esEs":{"MENU":{"CART":{"EMPTY":"Vaciar Carrito","CHECKOUT":"Realizar Pedido"},"USER":{"LABEL":"Usuario","DROPDOWN":{"EDIT":"Editar","LOGOUT":"Finalizar la Sesión"}}},"COUNTRY":{"USA":{"STATES":{"CALIFORNIA":{"NAME":"California","ACRONYM":"CA"},"NEW_YORK":{"NAME":"Nueva York","ACRONYM":"NY"},"NORTH_CAROLINA":{"NAME":"Carolina del Norte","ACRONYM":"NC"}}}}}}